src/pages/Docchi/main.ts

```diff
@@ -0,0 +1,109 @@
+import { pageInterface } from '../pageInterface';
+
+export const Docchi: pageInterface = {
+  name: 'Docchi',
+  domain: 'https://docchi.pl',
+  languages: ['Polish'],
+  type: 'anime',
+  isSyncPage(url) {
+    if (url.split('/')[5]) {
+      return true;
+    }
+    return false;
+  },
+  sync: {
+    getTitle(url) {
+      return j.$('a[mal_sync="title"]').text() || '';
+    },
+    getIdentifier(url) {
+      return url.split('/')[4];
+    },
+    getOverviewUrl(url) {
+      return utils.absoluteLink(j.$('a[mal_sync="title"]').attr('href'), Docchi.domain);
+    },
+    getEpisode(url) {
+      return Number(j.$('a[mal_sync="episode"]').text());
+    },
+    nextEpUrl(url) {
+      const href = j.$('a[mal_sync="title"]').attr('href');
+      const episode = Number(j.$('a[mal_sync="episode"]').text()) + 1;
+
+      return utils.absoluteLink(`${href}/${episode}`, Docchi.domain);
+    },
+    getMalUrl(provider) {
+      return `https://myanimelist.net/anime/${j
+        .$('a[mal_sync="episode"]')
+        .attr('mal_sync_mal_id')}`;
+    },
+  },
+  overview: {
+    getTitle(url) {
+      return j.$('a[mal_sync="title"]').text();
+    },
+    getIdentifier(url) {
+      return Docchi.sync!.getIdentifier(url);
+    },
+    uiSelector(selector) {
+      j.$('div[mal_sync="episodes_list"]').before(j.html(selector));
+    },
+    getMalUrl(provider) {
+      return `https://myanimelist.net/anime/${j
```

Comment:
Does not work resulting in a error and nothing works. Please add a condition.

Reply:
I guess you was mentioning `getMalUrl`, I fixed it now returns ID perfectly

Reply:
Please an condition for animes that have no mal id.